### ISSUE TITLE
`MissingTemplate` reports error for file that's ignored in `.theme-check.yml`

### ISSUE THREAD

**Issue by vfonic** [Shopify/theme-check#589]
1. Add this to `.theme-check.yml`:
  ```yml
ignore:
  - snippets/missing.liquid
  ```
2. Add `snippets/missing.liquid` file to your project
3. Add `{% render 'missing' %}` to some liquid file

Expected:

No errors reported

Actual:

Error reported:

> 'snippets/missing.liquid' is not found (theme-check MissingTemplate)
